package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@tracktor/react-utils",
   "description": "React data table and react data grid",
-  "version": "1.24.0",
+  "version": "1.25.0",
```

On va pas faire une version de feature pour mais plutôt de fix 
```suggestion
  "version": "1.24.1",
```